docs/user/json_data_converter_HOWTO.md

```diff
@@ -0,0 +1,536 @@
+        value_paths: [value.Value.StringValue]
+        value_type: string
+    job_id:
+        key_paths: [resource_spans.[].scope_spans.[].spans.[].trace_id]
+        value_type: string
+    event_id:
+        key_paths: [resource_spans.[].scope_spans.[].spans.[].span_id]
+        value_type: string
+```
+
+This will return the following output records for the [JSON example](#json-example):
+```json
+{
+    "job_name": "Test App",
+    "job_id": "trace001",
+    "event_id": "span001"
+}
+{
+    "job_name": "Test App",
+    "job_id": "trace002",
+    "event_id": "span002"
+}
+```
+
+### Example 4: Example for obtaining the full OTelEvent schema (wihtout child_event_ids)
+In this example the user wants to extract all the fields in the OTelEvent schema except for `child_event_ids` which is not present in the JSON data.
+
+```yaml
+field_mapping:
+    job_name:
+        key_paths: [resource_spans.[].resource.attributes.[].key]
+        key_value: [service.name]
+        value_paths: [value.Value.StringValue]
+        value_type: string
+    job_id:
+        key_paths: [resource_spans.[].scope_spans.[].spans.[].trace_id]
+        value_type: string
+    event_type:
+        key_paths: [
+            resource_spans.[].scope_spans.[].spans.name,
+            [
+                resource_spans.[].scope_spans.[].spans.not_here, # this is not a valid path
+                resource_spans.[].scope_spans.[].spans.attributes.[].key # this is the fallback path
+            ]
+        ]
+        key_value: [null, [null, http.response]]
+        value_paths: [null, [null, value.Value.IntValue]]
+        value_type: string
+    event_id:
+        key_paths: [resource_spans.[].scope_spans.[].spans.[].span_id]
+        value_type: string
+    start_timestamp:
+        key_paths: [
+            resource_spans.[].scope_spans.[].spans.[].start_time_unix_nano
+        ]
+        value_type: string
+    end_timestamp:
+        key_paths: [
+            resource_spans.[].scope_spans.[].spans.[].end_time_unix_nano
+        ]
+        value_type: string
+    application_name:
+        key_paths: [resource_spans.[].scope_spans.[].scope.name]
+        value_type: string
+    parent_event_id:
+        key_paths: [resource_spans.[].scope_spans.[].spans.[].parent_span_id]
+        value_type: string
+```
+
+This will return the following output records for the [JSON example](#json-example):
+```json
+{
+    "job_name": "Test App",
+    "job_id": "trace001",
+    "event_type": "/delete_200",
+    "event_id": "span001",
+    "start_timestamp": "1723544132228102912",
+    "end_timestamp": "1723544132228219285",
+    "application_name": "Group 1",
+    "parent_event_id": null
+}
+{
+    "job_name": "Test App",
+    "job_id": "trace002",
+    "event_type": "/put_200",
+    "event_id": "span002",
+    "start_timestamp": "1723544132228102912",
+    "end_timestamp": "1723544132228219285",
+    "application_name": "Group 1",
+    "parent_event_id": null
```

Comment:
Think there should be a parent_span_id in the example JSON for span002, which should be span001

Reply:
Wanted a case where that wasn't pulled out